source/MaterialXGenShader/ShaderNode.cpp

```diff
@@ -342,21 +347,48 @@ ShaderNodePtr ShaderNode::create(const ShaderGraph* parent, const string& name,
     return newNode;
 }
 
-void ShaderNode::setPaths(const Node& node, const NodeDef& nodeDef, bool includeNodeDefInputs)
+void ShaderNode::initialize(const Node& node, const NodeDef& nodeDef, GenContext& context)
```

Comment:
Not sure here: What happens to the `includeNodeDefInputs` ?

Reply:
It was never used. Its default value (true) was never changed in any usage of this function that I could find. So this was just a small code cleanup.